### PR TITLE
Some more info on available directories for .lex extensions

### PR DESCRIPTION
**Pull request by paulklinkenberg**

### CODE REVIEW

File: docs/04.guides/08.lucee-5/01.extensions/page.md [hunk 1]
```diff
@@ -30,9 +30,30 @@ You can install extensions by declaring the "id" of the extension as a JVM argum
...
-/applications - for cfm & cfc files that need to be installed in the webroot
-/functions - for build-in functions to install
-/jars - for libraries used by your extension
+- /applications - for cfm & cfc files that need to be installed in the webroot
+- /functions - for built-in functions to install
+- /jars - for libraries used by your extension
+- /plugins/{plugin name}/* - for adding an admin plugin
+
+The complete list of directories and file extensions which you can add to a .lex archive is (distilled from [the source code](https://github.com/lucee/Lucee/blob/master/core/src/main/java/lucee/runtime/config/XMLConfigAdmin.java#L4527)):
+`/jars/*.jar`
```
Comment (DominicWatson) [inline docs/04.guides/08.lucee-5/01.extensions/page.md]: Might be good to make this a bullet list. These single newlines in .md will actually result in one big paragraph that's not so readable.

**Comment by DominicWatson** [comment]
Great stuff :) Are you able to make change commented on above? (or argue against it ;) ). Can merge straight in then.

**Comment by paulklinkenberg** [comment]
Hi Dominic, as you can see [here](https://github.com/paulklinkenberg/lucee-docs/blob/patch-1/docs/04.guides/08.lucee-5/01.extensions/page.md), the first list is shown as a bullet list, and the other list is indeed all on the same line, but all items have a grey background, and seperated by whitespace.
This was on purpose, because the list was quite long, and only of interest to some of the readers.

**Comment by andrew-dixon** [comment]
@DominicWatson @paulklinkenberg I've been editing some documentation today that @micstriit has produced for extensions in Lucee 5, and from a very quick look this might duplicate some of that, so I will keep this in mind when I'm adding the new documentation once Micha has ok'ed it.

**Comment by DominicWatson** [comment]
> This was on purpose, because the list was quite long, and only of interest to some of the readers.

Fair enough, I'll keep it as is. Merging (apologies for the delay getting this in).

**Comment by paulklinkenberg** [comment]
Thanks :)

**Pull request merged**